scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -569,7 +569,15 @@ public void copyScratchDataToOutput(List<OutputSelector> outputFiles) throws Fil
     }
 
     @Override
-    public void cleanScratchSpace() {
+    public void close() {
+        if (!executorTransfer.shutdownNow().isEmpty()) {
+            logger.error("Remaining tasks to execute while closing thread pool used for data transfer");
```

Comment:
The error is logged but not acted on. So can it be that there are still files to transfer and the Dataspace is closed? Can't we "notify" the calling code about the inconsistency?